app/reducers/views/channel.js

```diff
@@ -199,10 +200,54 @@ function tooltipVisible(state = false, action) {
     }
 }
 
+function postVisibility(state = {}, action) {
+    switch (action.type) {
+    case ChannelTypes.SELECT_CHANNEL: {
+        const nextState = {...state};
+        nextState[action.data] = Posts.POST_CHUNK_SIZE / 2;
```

Comment:
Regarding my first comment, if we want to consistently use a half chunk size like this, maybe we can make it into another constant like `POST_VISIBILITY_CHUNK_SIZE`